core/aws-lsp-core/src/util/workspaceUtils.ts

```diff
@@ -12,14 +12,16 @@ export async function readDirectoryRecursively(
     folderPath: string,
     options?: {
         maxDepth?: number
-        excludeEntries?: string[]
+        excludeDirs?: string[]
+        excludeFiles?: string[]
         customFormatCallback?: (entry: Dirent) => string
         failOnError?: boolean
     },
     token?: CancellationToken
 ): Promise<string[]> {
     const dirExists = await features.workspace.fs.exists(folderPath)
-    const excludeEntries = options?.excludeEntries ?? []
+    const excludeFiles = options?.excludeFiles ?? []
+    const excludeDirs = options?.excludeDirs ?? []
```

Comment:
is the listDirectory use the same workspaceUtils.ts, do we need to update the listDirectory?

Reply:
it does not use the same method. It uses a new method that is already updated